channels/views/channels_test.py

```diff
@@ -424,6 +425,32 @@ def test_patch_channel_forbidden(staff_client):
     assert resp.status_code == status.HTTP_403_FORBIDDEN
 
 
+@pytest.mark.parametrize("request_method", ["patch", "put"])
+def test_update_title_non_superuser_forbidden(
+    staff_user, client, public_channel, request_method
+):
+    """
+    Non-superusers should be forbidden from updating a channel's title
+    """
+    staff_user.is_superuser = False
+    staff_user.save()
+    client.force_login(staff_user)
+    url = reverse("channel-detail", kwargs={"channel_name": public_channel.name})
+    serialized_channel = client.get(url).json()
+    # HACK: Manually setting these required values to satisfy validation on PUT requests
+    serialized_channel.update({"avatar": "", "banner": "", "ga_tracking_id": ""})
```

Comment:
@mbertrand Included this here because I couldn't find any existing fixtures or straightforward ways to create an object that would be ready to submit as `PUT` and `PATCH` requests to update a channel. Ended up taking much more time than I was comfortable with so I settled for this. I consider it pretty low risk